dockstore-common/src/main/java/io/dockstore/common/PipHelper.java

```diff
@@ -14,7 +14,8 @@
  */
 public final class PipHelper {
     public static final String DEV_SEM_VER = "development-build";
-    public static final String SEM_VER_STRING = "(^(\\d+)\\.(\\d+)\\.(\\d+)$)|(^" + DEV_SEM_VER + "$)";
+    // Regex copied from https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
```

Comment:
Now that you've modified the regex to handle catastrophic backtracking per Walt's comment, can you please update the comment ("based on with fix for REDOS", or something like that).